test/environmentDetails/EnvironmentDetailsHeader.test.tsx

```diff
@@ -70,9 +70,30 @@ describe("<EnvironmentDetailsHeader />", () => {
       store.dispatch(modeChanged(EnvironmentDetailsModes.CREATE));
     });
 
-    const input = component.getByPlaceholderText("Environment name");
+    const input = component.getByLabelText("Environment name");
     const newEnvName = "My new environment name";
     fireEvent.change(input, { target: { value: newEnvName } });
     expect(mockOnUpdateName).toHaveBeenCalledWith(newEnvName);
+    expect(component.queryByText("Namespace")).not.toBeInTheDocument();
+  });
+
+  it("should render component in create mode with namespace", () => {
+    const mockOnUpdateName = jest.fn();
+    const component = render(
+      mockTheme(
```

Comment:
For my own understanding: why use `mockTheme` instead of `<MockTheme>` for that component here?

Reply:
Good question. Two things:

1. I'm not super familiar with MUI or testing MUI. I was just patterning after what's already in this test file.
2. The function `mockTheme` is not written as a functional React component, but it could be.

The utility function is so lightweight that I'm thinking it might be better to replace the call to mockTheme() everywhere with:

```js
import { condaStoreTheme } from "../src/theme";
// ...
<ThemeProvider theme={condaStoreTheme}>
  <Provider store={store}>
    <ComponentUnderTest />
  </Provider>
</ThemeProvider>
```